lib/connectors/openid.js

```diff
@@ -27,6 +27,11 @@ module.exports = function (openIdConfiguration, scimConfiguration = {}) {
   });
 
   return {
+    /**
+     * this callback is registered considering the xblog npm module.
+     * we need this callback to be called on every authroize function called.
+     */
+    cb: cb,
```

Comment:
Please use a relevant name here. It can be an interceptor or simply a callback. Name should indicate the purpose this solves.

Reply:
i have added comment for the same purpose .

Reply:
That is not enough. Please use a contextual name.